docs/code-comparisons/kedro.rst

```diff
@@ -0,0 +1,248 @@
+  ) -> pd.DataFrame:
+      """Combines all data to create a model input table."""
+      shuttles = shuttles.drop("id", axis=1)
+      model_input_table = shuttles.merge(
+          companies, left_on="company_id", right_on="id"
+      )
+      model_input_table = model_input_table.dropna()
+      return model_input_table
+
+  if __name__ == "__main__":
+      from hamilton import driver
+      import dataflow  # import itself as a module
+
+      dr = driver.Builder().with_modules(dataflow).build()
+      inputs=dict(
+          companies=pd.read_parquet("path/to/companies.parquet"),
+          shuttles=pd.read_parquet("path/to/shuttles.parquet"),
+      )
+      results = dr.execute(["model_input_table"], inputs=inputs)
+
+
+Framework weight
+----------------
+
+After imperative vs. declarative, the next largest difference is the type of user experience they provide. ``Kedro`` is a more opiniated and heavier framework; ``Hamilton`` is on the opposite end of the spectrum and tries to be the lightest library possible. This changes the learning curve, adoption, and how each tool will integrate with your stack.
+
+Kedro
+~~~~~
+``Kedro`` is opiniated and provides clear guardrails on how to do things. To begin using it, you'll need to learn to:
+
+- Define nodes and register pipelines
+- Register datasets using the data catalog construct
+- Pass parameters to data runs
+- Configure environment variables and credentials
+
+This provides guidance when building your first data pipeline, but it's also a lot to take in at once. As you'll see in the `project comparison on GitHub <https://github.com/DAGWorks-Inc/hamilton/tree/main/examples/kedro>`_, ``Kedro`` involves more files making it harder to navigate. Also, it's reliant on YAML which is `generally seen as an unreliable format <https://noyaml.com/>`_. If you have an existing data stack or favorite library, it might clash with ``Kedro``'s way of thing (e.g., you have credentials management tool; you prefer `Hydra <https://hydra.cc/>`_ for configs).
+
+Hamilton
+~~~~~~~~
+
+``Hamilton`` attempts to get you started quickly. In fact, this page pretty much covered what you need to know:
+
+- Define nodes and a dataflow using regular Python functions (no need to even import ``hamilton``!)
+- Build a ``Driver`` with your dataflow module and call ``.execute()`` to get results
+
+``Hamilton`` allows you to start light and opt-in features as your project's requirements evolve (data validation, scaling compute, testing, etc.). Python is a powerful language with rich editor support and tooling hence why it advocates for "everything in Python" instead of external configs in YAML or JSON. For example, parameters, data assets, and configurations can very much live as dataclasses within a ``.py`` file. ``Hamilton`` was built with an extensive plugin system. There are many extensions, some contributed by users, to adapt Hamilton to your project, and it's easy for you to extend yourself for further customization.
```

Comment:
```suggestion
``Hamilton`` allows you to start light and opt-in features as your project's requirements evolve (data validation, scaling compute, testing, etc.). Python is a powerful language with rich editor support and tooling hence why it advocates for "everything in Python" instead of external configs in YAML or JSON. For example, parameters, data assets, and configurations can very much live as dataclasses within a ``.py`` file. ``Hamilton`` was built with an extensive plugin system. There are many extensions, some contributed by users, to adapt Hamilton to your project, and it's easy for you to extend yourself for further customization.

Note: Hamilton is so lightweight, you could even run it within Kedro!
```